src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/matrix_nms_ref.cl

```diff
@@ -0,0 +1,325 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/common.cl"
+#include "include/batch_headers/data_types.cl"
+
+#define NUM_BATCHES INPUT0_BATCH_NUM
+#define NUM_BOXES   INPUT0_FEATURE_NUM
+#define NUM_CLASSES INPUT1_FEATURE_NUM
```

Comment:
could you undef these at the end of file

Reply:
done